src/chunk.c

```diff
@@ -2723,11 +2723,12 @@ ts_chunks_in(PG_FUNCTION_ARGS)
 	pg_unreachable();
 }
 
-/* has chunk, specifieid by chunk_id, been compressed */
+/* Check if this chunk can be compressed, that is is not dropped and has not
```

Comment:
This changes the meaning of what was there before. This function used to check if the chunk is already compressed, not that it can be compressed.

Reply:
Al
```suggestion
/* Check if this chunk can be compressed, that it is not dropped and has not
```
```suggestion
/* Check if this chunk can be compressed, that is is not dropped and has not
```

Reply:
Yes, but it is not used elsewhere and it is renamed, so I think this is a good change.

Reply:
Type still not fixed.

Reply:
Done.